cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,10 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('/#/login?_k=7d9zy5');
+    cy.get('input[placeholder="Email"]').type('testing123qwer@gg.com');
+    cy.get('input[placeholder="Password"]').type('testing123');
+    cy.get('button').contains('Sign in').click();
+    cy.get('.nav-link a').should('contain.text', 'testing123qwer');
```

Comment:
There is no such element as `a` inside the element `.nav-link`. If you wanted to say "element a with the class nav-link" you should select `a.nav-link`.

Reply:
fixed